draft-ietf-quic-http.md

```diff
@@ -389,6 +389,23 @@ HTTP_PUSH_ALREADY_IN_CACHE; see {{errors}}).  This asks the server not to
 transfer the data and indicates that it will be discarded upon receipt.
 
 
+## Solicited Stream Cancellation
+
+If an endpoint is no longer interested in the data being received, it MAY send a
+STOP_SENDING frame to request prompt closure of the stream in the opposite
+direction.  This typically indicates that the receiving application is no longer
+reading from the stream, but does not guarantee that incoming data will be
+ignored.
+
+Upon receipt of a STOP_SENDING frame on a stream in the "open" or "half-closed
+(remote)" states, an endpoint MUST send a QUIC RST_STREAM frame.  If the
+STOP_SENDING frame is received on a stream that is already in the "half-closed
+(local)" or "closed" states, a RST_STREAM frame MAY still be sent in order to
+cancel retransmission of previously-sent STREAM frames.  Sending a STOP_SENDING
```

Comment:
Moving this piece to the HTTP layer adds an implicit new feature to QUIC API surfaces -- the ability to RST a closed stream.